plugins/module_utils/base.py

```diff
@@ -52,3 +53,9 @@ def grafana_required_together():
 
 def grafana_mutually_exclusive():
     return [["url_username", "grafana_api_key"]]
+
+
+def dict_str(value):
+    if isinstance(value, (dict, string_types)):
+        return value
+    raise TypeError
```

Comment:
Moving `dict_str` to module_utils is fine, but not absolutely necessary, since this function isn't used multiple times.

Reply:
I did it as maybe other parts could use it as well. OK to keep?